openstudiocore/src/model/AirLoopHVAC_Impl.hpp

```diff
@@ -308,4 +304,4 @@ class MODEL_API AirLoopHVAC_Impl : public Loop_Impl {
 
 } // openstudio
 
-#endif
+#endif
```

Comment:
@jmarrec let's avoid these warnings about no new line at end of file

Reply:
Sorry, for what it's worth I didn't do that on purpose: I removed the `Quantity`-related methods using python, basically `file.read().splitlines()` then `file.write("\".join(new_lines))`, which has the effect of removing the last `\n`...